Formula/rtl_433.rb

```diff
@@ -0,0 +1,41 @@
+class Rtl433 < Formula
+  desc "Program to decode radio transmissions from devices"
+  homepage "https://github.com/merbanan/rtl_433"
+  url "https://github.com/merbanan/rtl_433.git",
+    tag:      "20.11",
+    revision: "0bb4672b1afb716d107b3060afc2d80a2159dbef"
```

Comment:
```suggestion
      tag:      "20.11",
      revision: "0bb4672b1afb716d107b3060afc2d80a2159dbef"
```
Let's follow the way other formulae align the `url` arguments.